March 24 Product Updates (#29)
greatexpectationslabs/workshops: get_started_with_gx_cloud/workshop.md

> **Select the `nyc_taxi_data` Data Asset**
> 1. On the **Select tables to import** page, check the box next to `nyc_taxi_data`.
> 1. Click **Add 1 Asset**.
> 1. Click **Start monitoring**.

<img src="img/add_demo_data_asset.png" alt="Add a Demo Data Asset" style="width:600px;"/><br>

> <img src="img/add_demo_data_monitoring.png" alt="Start Monitoring" style="width:600px;"/><br>

Reviewer: This alt text repeats the button text that's already referenced in the preceding prose. Here's a suggestion that communicates more of what's visually available in the screenshot

```suggestion
<img src="img/add_demo_data_monitoring.png" alt="By default, GX Cloud automatically generates Expectations to detect schema and volume issues. You can de-select recommendations you’d like to opt out of." style="width:600px;"/><br>
```

> Congratulations! You have successfully added the `nyc_taxi_data` demo Data Asset to your GX Cloud organization. Click onto the newly-created Data Asset to view it.

Reviewer: "Click onto" is unusual phrasing we don't use elsewhere. Suggestion for more standard phrasing:

```suggestion
Congratulations! You have successfully added the `nyc_taxi_data` demo Data Asset to your GX Cloud organization. Click on the newly-created Data Asset to view it.
```

> <img src="img/view_demo_asset.png" alt="View Demo Data Asset" style="width:600px;"/><br>

Reviewer: Similar to another comment, here's a suggestion for more meaningful alt text

```suggestion
<img src="img/view_demo_asset.png" alt="The asset has default coverage for schema and volume. There are calls to action to validate the Data Asset and add completeness coverage." style="width:600px;"/><br>
```

> ## Generate Expectations

Reviewer: The links in the "Agenda" section above should be updated to reflect this additional task and the changed name of the next task

> GX Cloud helps you get started on your data quality journey by leveraging AI to analyze your data and generate Expectations for you. This is especially helpful if you're just getting started and are unsure of where to start with your data quality checks. 

Reviewer: re "This is especially helpful if you're just getting started and are unsure of where to start with your data quality checks." I'm wary of highlighting lack of user experience as the main benefit for this feature. Could this be reworkd to highlight time-to-value for all users regardless of level of experience instead?

> > 1. Click on the **Generate Expecations** button.

Reviewer: typo

```suggestion
> 1. Click on the **Generate Expectations** button.
```

> <img src="img/generate_expectations.png" alt="Generate Expectations" style="width:600px;"/><br>

Reviewer: similar to another comment, here's a suggestion for more meaningful alt text

```suggestion
<img src="img/generate_expectations.png" alt="The button for this Beta feature is between the buttons for Alerts and New Expectation." style="width:600px;"/><br>
```

> GX Cloud will work in the background to generate Expectations, leveraging the power of AI. The generated Expectations may be different each time you use the tool. To see other iterations, simply delete the Expectations that were created and click on the **Generate Expectations** button to try again.

Reviewer: I suggest dropping the word "simply" for concision. Also, this word may backfire and frustrate folks who would prefer to bulk-delete generated Expectations instead of having to delete them each individually. 
```suggestion
GX Cloud will work in the background to generate Expectations, leveraging the power of AI. The generated Expectations may be different each time you use the tool. To see other iterations, delete the Expectations that were created and click on the **Generate Expectations** button to try again.
```

> <img src="img/delete_generated_expectations.png" alt="Delete Generated Expectations" style="width:600px;"/><br>

Reviewer: similar to another comment, here's a suggestion for more meaningful alt text

```suggestion
<img src="img/delete_generated_expectations.png" alt="Delete Expectations one-by-one. Each Expectation has a Delete Expectation button." style="width:600px;"/><br>
```
